### PR TITLE
Fix plugins prioritization order information

### PR DESCRIPTION
**Pull request by rodrigowebjump**
## Purpose of this pull request

This pull request (PR) fix the explanation about prioritization and execution order flow and includes additional examples to better understanding.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.2/extension-dev-guide/plugins.html
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html

whatsnew
Added substantial new information about plugin execution order in [Plugins (Interceptors)](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.2/extension-dev-guide/plugins.md [hunk 1]
```diff
@@ -290,50 +290,132 @@ class MyUtilityUpdater
...
+
+The execution will have a different flow, depending on the methods implemented by these classes, as explained in the following scenarios. 
+
+**Scenario A**
+
+Your plugin classes has this methods: 
+
+|               | PluginA          | PluginB          | PluginC          |
+| :-----------: | :--------------: | :--------------: | :--------------: |
+| **before**    | beforeDispatch() | beforeDispatch() | beforeDispatch() |
```
Comment (aleron75) [inline src/guides/v2.2/extension-dev-guide/plugins.md]: I think the sortOrder row was useful, what about re-introducing it? The same for other tables.

**Comment by dobooth** [comment]
Hi @rodrigowebjump Thanks for the submission. What do you think of the feedback from @aleron75 ?

**Comment by rodrigowebjump** [comment]
Hi @aleron75 

I just updated the examples.
Please, could you review it?

Regards

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @rodrigowebjump

**Pull request merged**